src/frame/endpoints/tests/GroupApiEndpoints.test.ts

```diff
@@ -280,7 +281,7 @@ describe("GroupApiEndpoints", () => {
                 },
             ];
 
-            GroupApiEndpoints.callAddAdminsApi("22", userKeys).engage(
+            GroupApiEndpoints.callAddAdminsApi("22", userKeys, signature).engage(
```

Comment:
You should add the value of this `signature` into the body assertion below.

Reply:
done